doc/api/util.markdown

```diff
@@ -548,6 +548,16 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
+## util._extend(obj)
+
+    Stability: 0 - Deprecated: Use Object.assign() instead.
+
+`_extend` was never intended to be used outside of internal NodeJS modules. The
```

Comment:
I think we conveyed this at the top of the page. https://nodejs.org/api/util.html#util_util